pages/index.vue

```diff
@@ -67,10 +61,20 @@
 			Footer,
             Button
         },
+        async asyncData(context) {
+            let redirectUrl
+
+            try {
+                redirectUrl = await context.$axios.$get('auth/discord/redirect')
+            } catch (error) {
+                console.error(error)
+            }
+
+            return { redirectUrl }
+        },
```

Comment:
I think this function is better as a promise. 
```suggestion
        asyncData = (context) => new Promise((resolve, reject) => {
            context.$axios.$get('auth/discord/redirect')
                .then(result => resolve(result))
                .catch(reason => reject(reason))
        })
```

Reply:
I think it works fine like it is, and it doesn't really adds a lot of load, plus Vue doesn't seems to like using a Promise on this case.